RELEASES.md

```diff
@@ -1,3 +1,105 @@
+Version 1.30.0 (2018-10-25)
+==========================
+
+Language
+--------
+- [This version marks the release of the 2018 edition of Rust.][54057]
```

Comment:
I thought we were releasing Rust 2018 in December?

Reply:
Technically it's stable on master (soon to be beta)?

Reply:
See https://internals.rust-lang.org/t/rust-2018-release-schedule-and-extended-beta/8076.
First stable release of Rust 2018 should be in 2018-12-06 according to that.
cc @aturon

Reply:
We should probably reword a bit to reflect this :)